### DISCUSSION TITLE
Is there anyway to change add fonts for the terminal, where the code-server is hosted by termux.

### DISCUSSION THREAD

**Discussion by MMMMMoris** [coder/code-server#5880] [Q&A]
Is there anyway to change or add fonts for the terminal, where the code-server is hosted by termux?
Here is my code-server while browsing

![Screenshot_2022-12-14-04-01-49-766_com android chrome](https://user-images.githubusercontent.com/63332824/207449296-73ddcc97-deca-4d4c-8339-c6371b3d867a.jpg)

It's installed on Termux with proot-distro and ubuntu. However, the powerlevel10k terminal prompt cannot display its fonts normally
The fonts display normally on the terminal of termux as well as ubuntu, except the terminal on code-server...
Any solution will be appreciated.

**Comment by jsjoeio** [comment]
So they're installed on your device or rather Termux? Hmm you should be able to use them. Have you tried changing the font family in your settings? 

```json
"terminal.integrated.fontFamily": "Hack Nerd Font, Menlo, Monaco, 'Courier New', monospace"
```
